imap_processing/cdf/utils.py

```diff
@@ -48,13 +49,18 @@ def calc_start_time(
     return launch_time + time_delta
 
 
-def load_cdf(file_path: Path, **kwargs: dict) -> xr.Dataset:
+def load_cdf(
+    file_path: Path, remove_xarray_attrs: bool = True, **kwargs: dict
```

Comment:
Are you going to add `to_datetime=True` in another PR? Just curious

Reply:
Oh... thanks for reminding me. I'll add that here.

Reply:
I have added this change to this PR. @tech3371 and @greglucas, can you take another look at this PR for those changes. I wasn't sure if my implementation of setting that default is too opaque and should be done through an explicit keyword argument.

Reply:
I don't think this is in the `cdflib` upgrade. @bryan-harter Can you confirm? I think `cdflib` still keeps this as optional parameter. But Bryan mentioned this in our Slack which is little different from this. Is that what you are thinking?
```
Going along with the above, "datetime64_to_tt2000" is no longer needed in the xarray_to_cdf function. I decided to just make that the default behavior because why else would you have a datetime object if you didn't want a CDF time variable
```